src/NuGet.Status/Global.asax.cs

```diff
@@ -21,26 +22,30 @@ namespace NuGet.Status
 {
     public class MvcApplication : System.Web.HttpApplication
     {
-        public static IConfigurationProvider ConfigurationProvider;
+        private static TimeSpan ConfigurationRefreshPeriod = new TimeSpan(1, 0, 0, 0);
```

Comment:
nit: is this one day? add it as a comment.

Reply:
better yet, parameter names :)

Reply:
Yes, one day. Will add comment.